monkestation/code/modules/virology/disease/base_disease_folder/_base.dm

```diff
@@ -242,3 +242,47 @@ GLOBAL_LIST_INIT(virusDB, list())
 
 
 	ticks += speed
+
+//horrible, awful, stolen code from disease/advance. But it WORKS
+/datum/disease/acute
+	var/list/properties = list()
+
+// Calling A_Refresh effectivley checks for, then sets the severity
+
+/datum/disease/acute/proc/A_Refresh(new_name = FALSE)
+	A_GenerateProperties()
+	A_assign_properties()
+
+/datum/disease/acute/proc/A_GenerateProperties()
+	properties = list("severity" = 0)
+	for(var/datum/symptom/S in symptoms)
+		if(!S.neutered)
+			properties["severity"] = max(properties["severity"], S.severity) // severity is based on the highest severity non-neutered symptom
+
+/datum/disease/acute/proc/A_assign_properties()
+	if(properties?.len)
+		A_set_severity(properties["severity"])
+	else
+		CRASH("Our properties were empty or null!")
+
+
+/datum/disease/acute/proc/A_set_severity(level_sev)
```

Comment:
I'd prefer these have more descriptive names (and maybe documentation, but I'm not gonna be picky), alongside just being snake\_case in general

Reply:
also, change `properties?.len` to `length(properties)`